server/member/leader.go

```diff
@@ -399,6 +416,20 @@ func (m *Member) SetMemberBinaryVersion(id uint64, releaseVersion string) error
 	return nil
 }
 
+// SetMemberGitHash saves a member's git hash.
+func (m *Member) SetMemberGitHash(id uint64, gitHash string) error {
+	key := m.getMemberGitHashPath(id)
+	txn := kv.NewSlowLogTxn(m.client)
+	res, err := txn.Then(clientv3.OpPut(key, gitHash)).Commit()
```

Comment:
Why do we need persistence, I understand the version has been saved in the binary file?

Reply:
The version is not equal to version, and here we need git_hash. Is there any way to not save git_hash and allowing us to get it?